### PR TITLE
Add test cases for function node

### PR DESCRIPTION
**Pull request by kazuhitoyokoi**
I created test cases for function node to improve the code coverage.

In the test cases, keys() in [node context](https://github.com/kazuhitoyokoi/node-red/commit/8c6a270e80c667412844a3b2c2c51f8b68a63471#diff-9a05b775cd70a3070353c2c4bee12db0R312) and [flow context](https://github.com/kazuhitoyokoi/node-red/commit/8c6a270e80c667412844a3b2c2c51f8b68a63471#diff-9a05b775cd70a3070353c2c4bee12db0R376) return their method names only. But keys() in [global context](https://github.com/kazuhitoyokoi/node-red/commit/8c6a270e80c667412844a3b2c2c51f8b68a63471#diff-9a05b775cd70a3070353c2c4bee12db0R440) returns get, set, keys and their method names. Could you confirm if it is correct specification?

### CODE REVIEW

**Comment by kazuhitoyokoi** [comment]
@knolleary Thank you for your check. I'll survey the bug and submit another pull request.

File: test/nodes/core/core/80-function_spec.js [hunk 1]
```diff
@@ -228,6 +243,267 @@ describe('function node', function() {
...
+            n2.on("input", function(msg) {
+                msg.should.have.property('topic', 'bar');
+                msg.should.have.property('payload', '0');
+                done();
+            });
+            n1.receive({payload:"foo",topic: "bar"});
+        });
+    });
+
+    it('should get keys in global context', function(done) {
```
Comment (knolleary) [inline test/nodes/core/core/80-function_spec.js]: with #1417 merged, this test case needs removing I think?

**Comment by kazuhitoyokoi** [comment]
@knolleary I removed the duplicated test case. Please check it.

**Comment by knolleary** [comment]
thanks

**Comment by kazuhitoyokoi** [comment]
@knolleary Thank you for your check. 👍

**Pull request merged**